fastify.js

```diff
@@ -124,90 +126,111 @@ function build (options) {
   return fastify
 
   function fastify (req, res) {
-    logger(req, res)
+    req.id = genReqId(req)
+    req.log = res.log = logger.child({ req: req })
+    res[startTime] = res[startTime] || Date.now()
```

Comment:
I'm working on add a `onResponse` hook, that will run once the response has been sent.
To do that I must use a `response` event listener, so we used the same method of [`pino-http`](https://github.com/pinojs/pino-http).
@mcollina suggested to import directly the `pino-http` code inside Fastify, se here we are :)